ispyb/model/integration.py

```diff
@@ -85,3 +91,93 @@ def __str__(self):
     ('detector_distance', 'refinedDetectorDistance'),
     ('timestamp', 'recordTimeStamp'),
 ))
+
+class UnitCell():
+  '''An object representing the parameters of the unit cell I.e unit cell edges and angles
+  '''
+
+  def __init__(self, a,b,c,alpha,beta,gamma):
```

Comment:
Just a trivial code style thing: we generally put a space after every comma to make it easier to read. We mostly try to follow the [PEP-8 Python style guide](https://www.python.org/dev/peps/pep-0008/?#whitespace-in-expressions-and-statements) (with a few exceptions)